public/javascript/getWeather.js

```diff
@@ -8,9 +8,22 @@
 var weather = require("weather-js");
 
 module.exports = function getWeather(api, threadID, body) {
-    const locale = body.substring('@weather '.length);
-    console.log('Fetching weather for ' + locale + '...');
-    weather.find({ search: locale, degreeType: 'F' }, function(err, result) {
+    var locale = '';
+    var currentDType = 'F';
+    const infoArray = body.split(' ');
+    for (var i = 1; i < infoArray.length; i++) {
```

Comment:
This loop seems like a lot of work for just basically checking if the last thing is a C or an F.